src/devices/Common/CommonHelpers.csproj

```diff
@@ -3,11 +3,16 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
NIT we don't use RootNamespace in our projects so lets just clean it up from here

Reply:
@joperezr : Setting it to empty here just has the effect of automatically creating new files with the correct namespace, depending on their location in the directory tree. If this line is removed, the namespace of a file added to the "Common" folder will be something like `Iot.Common.Iot.Device.Common` instead of just `Iot.Device.Common`.